ui/src/app/base/components/ContextualMenu/ContextualMenu.js

```diff
@@ -9,7 +9,11 @@ const getPositionStyle = (position, wrapper, constrainPanelWidth) => {
   if (!wrapper || !wrapper.current) {
     return undefined;
   }
-  const { bottom, left, width } = wrapper.current.getBoundingClientRect();
+  const {
+    bottom,
+    left,
+    width
+  } = wrapper.current.children[0].getBoundingClientRect();
```

Comment:
Instead of accessing the children would it be possible to move the ref to the correct node?

Reply:
Hmm I can't pass it to the `Button` because you can only assign `ref`s to normal DOM elements, and there isn't currently a `useForwardRef` hook. I've added a guard in the function so that if a child isn't present it just gets the dimensions of the span, but otherwise it gets the first child's dimensions. Would that be ok for now?

Reply:
Yep, that's better I reckon.